src/main/tables/81_webhooks_subscription.sql

```diff
@@ -0,0 +1,9 @@
+SET search_path = public, pg_catalog;
+
+--
+-- webhooks_user_subscription table
```

Comment:
The table is named `webhooks_subscription`, but this comment says otherwise 😜

Reply:
Grr. Renamed the table but forgot to update comments :(